src/components/rails/explorer-rail/pmBlocksLoop/pmBlocksLoop.vue

```diff
@@ -41,8 +44,11 @@ export default {
             :key="nodeType.id"
             @click.stop="onClickHandler($event, nodeType)"
           >
-            <label>{{ $t(nodeType.label) }}</label>
-            <span class="d-block">{{ nodeType.description }}</span>
+            <div class="d-flex">
+              <i v-if="!svgIcon" class="node-types__item__icon" :class="nodeType.customIcon"/>
```

Comment:
@mcraeteisha  same as above `!object.svgIcon`

Reply:
Done! Thanks for catching.